### PR TITLE
Pertaining to issues #63 and #61

### PR DESCRIPTION
**Pull request by natasha-sharma123**

### CODE REVIEW

**Comment by pauldpickell** [comment]
@natasha-sharma123 I reviewed chapter 2. Not every image needs to have a URL linked directly to it in the caption. Only animated figures (GIFS) and leaflet maps should have this. See issue #61. For Chapter 2, I rejected most of your changes. I assume the same problem persists in the other chapters as well. Can you please carefully review each figure and ensure that you are only adding a direct URL for animated figures with a GIF extension or leaflet?

**Comment by natasha-sharma123** [comment]
> @natasha-sharma123 I reviewed chapter 2. Not every image needs to have a URL linked directly to it in the caption. Only animated figures (GIFS) and leaflet maps should have this. See issue #61. For Chapter 2, I rejected most of your changes. I assume the same problem persists in the other chapters as well. Can you please carefully review each figure and ensure that you are only adding a direct URL for animated figures with a GIF extension or leaflet?

Oh, I have fixed it now! Only chapter 2 and 5 had them but the rest don't have the figure caption for it so it should be good now. Also in chapter 15 line 353 is the animated figure caption required?

File: 12-remote-sensing-systems.Rmd [hunk 1]
```diff
@@ -323,15 +333,15 @@ _It's important to note that the "ground plane" refers to a plane tangent to the
 
 ```{r 12-aerial-good, fig.cap = fig_cap, out.width="90%", echo = FALSE}
 
-fig_cap <- paste("How aerial imagery should be taken. Claire Armour. Animated figure can be viewed in the web browser version of the textbook. [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)")
+fig_cap <- paste0("How aerial imagery should be taken. Claire Armour.<a href='",baseurl,"#fig:12-aerial-good'>Animated figure can be viewed in the web browser version of the textbook</a>. [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)")
```
Comment (ect123) [inline 12-remote-sensing-systems.Rmd:336]: Looks like a space needs to be added before the word "Animated"

**Review by ect123 — CHANGES_REQUESTED** [review]
@natasha-sharma123 this is almost complete - just a few small comments to look at before we can merge.

File: 09-raster-analysis-terrain-modelling.Rmd [hunk 2]
```diff
@@ -554,15 +562,15 @@ A **viewshed** is an analysis that simulates a physical line-of-sight using a DE
...
 
 ```{r 9-vancouver-population-density-2016, fig.cap = fig_cap, out.width= "75%", echo = FALSE}
-    fig_cap <- paste("Census dissemination areas extruded by population density (2016) for Vancouver, British Columbia. Animated figure can be viewed in the web browser version of the textbook. Census data from Statistics Canada, Census Profile, 2016, licensed under Statistics Canada Open License. Reproduced and distributed on an 'as is' basis with the permission of Statistics Canada. Base map © OpenStreetMap contributors, licensed under Open Data Commons Open Database License. Pickell, CC-BY-SA-4.0.")
+    fig_cap <- paste("Census dissemination areas extruded by population density (2016) for Vancouver, British Columbia. Census data from Statistics Canada, Census Profile, 2016, licensed under Statistics Canada Open License. Reproduced and distributed on an 'as is' basis with the permission of Statistics Canada. Base map © OpenStreetMap contributors, licensed under Open Data Commons Open Database License. Pickell, CC-BY-SA-4.0.")
```
[the remark this answers is not in the document]
Comment (natasha-sharma123) [reply 09-raster-analysis-terrain-modelling.Rmd]: All revisions are made and done now!

**Review by ect123 — APPROVED** [review]

**Pull request merged**